src/charm.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+
+"""SSSD Operator Charm."""
+
+import logging
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, BlockedStatus
+from sssd import SSSD
+
+logger = logging.getLogger(__name__)
+
+
+class SSSDCharm(CharmBase):
+    """SSSD Charm."""
+
+    def __init__(self, *args):
+        """Init observe events and sssd."""
```

Comment:
Only reason I added this was because the lint check kept complaining about it. I'll remove it and if it complains I'll see if I can find out how to ignore it